docs/platform/api/create-block-storage-volumes-with-the-linode-api.md

```diff
@@ -0,0 +1,197 @@
+        https://api.linode.com/v4/volumes
+
+    {{< note >}}
+The Volume and the Linode it will be attached to must be in the same region.
+{{< /note >}}
+
+2.  Examine the response JSON object and copy the new Volume's ID (in the `id` field).
+
+    {{< highlight json "linenos=table" >}}
+{
+   "linode_id":null,
+   "label":"my-volume",
+   "size":100,
+   "updated":"2018-05-07T14:59:48",
+   "created":"2018-05-07T14:59:48",
+   "id":6830,
+   "status":"creating",
+   "region":"us-east",
+   "filesystem_path":"/dev/disk/by-id/scsi-0Linode_Volume_my-volume"
+}
+{{< /highlight >}}
+
+3.   Query the Volume using the `/volumes/$volume_id` endpoint to to make sure it was successfully created:
+
+        curl -H "Authorization: Bearer $token" \
+        https://api.linode.com/v4/volumes/$volume_id
+
+      If the `status` field in the response is `active`, your Volume is ready to use.
+
+## Attach to a Linode
+
+Before the new Volume can be mounted, it must be attached to a Linode. To do this, you will need the ID of a Linode on your account.
+
+1.  List the Linodes on your account:
+
+        curl -H "Authorization: Bearer $token" \
+        https://api.linode.com/v4/linode/instances
+
+    Each Linode instance in the returned list will include an `id` field. Choose a Linode and copy its ID.
+
+2.  Attach the new Volume to your Linode with the `/volumes/$volume_id/attach` endpoint:
+
+        curl -H "Authorization: Bearer $token" \
+        -H "Content-Type: application/json" \
+        -X POST -d \
+        '{ "linode_id": $linode-id }' \
+        https://api.linode.com/v4/volumes/$volume_id/attach
+
+3.  Copy the file path in the `filesystem_path` field:
+
+    {{< highlight json "linenos=table" >}}
+{
+   "id":1234,
+   "status":"active",
+   "size":100,
+   "label":"my-volume",
+   "linode_id":1234567,
+   "region":"us-east",
+   "updated":"2018-05-07T17:22:27",
+   "created":"2018-05-07T17:22:27",
+   "filesystem_path":"/dev/disk/by-id/scsi-0Linode_Volume_my-volume"
+}
+{{< /highlight >}}
+
+    {{< note >}}
+If a Linode is not running and has more than one configuration profile, you should also include a `config_id` parameter in the POST request to specify which profile to use.
+{{< /note >}}
+
+### Mount the Volume
+
+The API can't directly mount the new Volume after it is attached; you will have to SSH into the Linode and mount manually:
+
+1.  Create a filesystem on the Volume:
+
+        mkfs.ext4 $volume_path
+
+2.  Create a mountpoint:
+
+        mkdir /mnt/my-volume
+
+3.  Mount the Volume:
+
+        mount $volume_path /mnt/my-volume
+
+4.  To have the Volume automatically mount every time your Linode boots, add the following line to your `/etc/fstab` file:
+
+        $volume_path /mnt/my-volume defaults 0 2
+
+### Detach
+
+To detach a Volume from your Linode, use the `/detach` endpoint:
+
+    curl -H "Authorization: Bearer $token" \
+    -X POST \
+    https://api.linode.com/v4/volumes/$volume_id/detach
+
+### Create and Attach
```

Comment:
Might consider this as the first example, since it's way more useful.. maybe the "Attach" example could show you how to move a volume from one Linode to another?